go/chat/attachments/store.go

```diff
@@ -88,7 +88,6 @@ type S3Store struct {
 	utils.DebugLabeler
 
 	g        *libkb.GlobalContext
```

Comment:
Sorry to keep bothering you with this stuff that isn't really the focus of this PR, but typically what happens here is we use the `libkb.Contextified` struct to provide a function called `G()` to the struct that includes it. If you grep around for that, there are some examples.

Reply:
Done.